tests/test_variables_script.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash  
+# this code will check if there is a variable without a default value in a given module, that is not define in the test for that module.
+# if the code find variables that is missing it will echo them back and the user will need to add them to the test for the module.
+
+cp $1 ./module_test_tmp.tf
+file="./module_test_tmp.tf"
+echo '' >> ./module_test_tmp.tf
```

Comment:
Better do `touch ./module_test_tmp.tf`

Reply:
I need to add a new empty line in the end of the file so that the code will read all of the file. When i tried to run it without an empty line in the end, the script didn't check the last variable in the variables.tf file, that is why i wanted to use echo instead of touch.